spec/support/tempfile.cr

```diff
@@ -1,7 +1,11 @@
 require "tempfile"
 require "file_utils"
 
-SPEC_TEMPFILE_PATH = File.join(Tempfile.dirname, "cr-spec-#{Random.new.hex(4)}")
+{% if flag?(:win32) %}
+  SPEC_TEMPFILE_PATH = File.join(Tempfile.dirname, "cr-spec-#{Random.new.hex(4)}").gsub("C:\\", '/').gsub('\\', '/')
```

Comment:
Why remove drive from path?

Reply:
Because `FileUtils.mkdir_p` can't handle windows paths.

Reply:
How about adding a TODO?

Reply:
It would ne nice to have something like `.gsub(/\A[a-zA-Z]:\\\\/)` which works with a temp path on other drives than `C:`.

Reply:
The TODO is to merge #5550. And honestly, I'm just hacking the specs around until they work here, then cleaning it up enough to be good to merge. At the end of it all we can just `grep -R 'flag\?(:win32' src` and find all this junk once we merge proper path handling support.